docs/howto/create-a-datasource/using-default-datasource.adoc

```diff
@@ -0,0 +1,16 @@
+
+[id='using-default-datasource_{context}']
+= Using the default datasource
+
+[discrete]
+== Prerequisites
+
+* A Maven-based application.
+* A JDBC Driver configured, either auto detected or manually.
+
+[discrete]
+== Procedure
+
+The presence of a JDBC driver will automatically create a datasource with the name `ExampleDS`.
```

Comment:
In the _Procedure_ sub-section, only a list of steps is allowed. The information you put here would make a good introduction, and instead you can put here an example of how to use the `thorntail.ds.name` property:

1. Customize the name of the datasource In the $CONFIG_FILE by adding the `thorntail.ds.name` property:
    EXAMPLE_CODE
2. Some other steps?

Reply:
By introduction, do you mean the text should be moved before the prerequisite section?

Reply:
Exactly. Possibly include there some more information like where using the default datasource is preferable or, on the other hand, where it is discouraged. Thanks.

Reply:
@tradej what's the way to say "put your value here"?

Is it like "{value}"?

Reply:
@kenfinnigan I'm not sure I quite understand the question, can you provide a little more context, please? `{value}` is a dereference of the attribute (i. e. AsciiDoc variable) named `value`.

Reply:
Basically how do I represent some yaml config like:

----
thorntail:
  ds:
    name: myDS
----

Where "myDS" is basically whatever name they want. Usually I'd make it something like "{yourName}" but not sure how we're meant to do that